insights/parsers/ssh.py

```diff
@@ -53,6 +53,10 @@
 from collections import namedtuple
 from .. import Parser, parser, get_active_lines
 from insights.specs import sshd_config
+import re
+
+# optional whitespace, at least one non-whitespace (the keyword), at least one whitespace (space), a plus literal, anything
+PLUS_PATTERN = re.compile(r'^\s*\S+\s+\+.*$')
```

Comment:
See below on why I think you should use named groups to provide the actual data from this value.

Reply:
I read the comments and I don't get it. I *think* that you mean that leaving the parsing of the individual values and stripping the leading "+" from the beginning of the first parsed value should be done by the parser rather than by the rule. In this interpretation then, modeling the line parsing through a regex with named groups would duplicate most of the parser's whole functionality. So, again continuing with that interpretation, I think it would make better sense to add a method that does the stripping of "+" or to extend the functionality of `get_values()`. However, I see `get_values()` as useless in my rules, since if I want it to split the values, it insists on joining all the lines with the given keyword - I rather use `last()` and split it myself, and that might be the source of our misunderstanding. Until `get_values()` behaves in a more fitting way, I'm not including the "+" parsing functionality. IIRC the testing I did back when I was beginning with my ssh rules, sshd, at least for some options, uses only the last line with the given keyword, which is why I do that in my rules. 

I understand your point about shared parsers preventing the same system file being read over and over again. It seems to me, however, that this parser tries to provide more functionality than what is necessary, and it complicates maintainability.

If I forewent with this MR, I could have done the "+" parsing solely in my rule and the parser would have been less correct and we wouldn't have this discussion. I agree with maintaining parsers well, but there's a business-practical balance to be met in my opinion.